build/generate-docs.ts

```diff
@@ -382,6 +382,9 @@ function createExpressionsContent() {
             }
             content += `\n### ${key}\n`;
             content += `${value.doc}\n`;
+            if ('example' in value) {
+                content += '\n`"some-propetry": ' + JSON.stringify(value.example) + '`\n';
```

Comment:
propetry, typo

Reply:
Thanks! I probably have a lot more typos here :-)
We'll fix them eventually...